src/Razor/test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/TagHelperBlockRewriterTest/AllowsCompatibleTagStructures8.cspans.txt

```diff
@@ -0,0 +1 @@
+Markup span at (7:0,7 [8] ) (Accepts:Any) - Parent: Tag block at (7:0,7 [8] )
```

Comment:
Just double checking, TagHelper elements don't get included in this list as of today correct?